libmamba/src/download/downloader.cpp

```diff
@@ -74,16 +84,62 @@ namespace mamba::download
                         LOG_INFO << "Using REQUESTS_CA_BUNDLE " << remote_fetch_params.ssl_verify;
                     }
                 }
-                else if (remote_fetch_params.ssl_verify == "<system>" && util::on_linux)
+                else if (remote_fetch_params.ssl_verify == "<system>")
```

Comment:
The semantic of `<system>` (which is the default) is unclear for now: must it uses conda-forge's distribution of CA certificates or must it use the system's?

To me conda-forge's distribution must be used by default either by being specified explicitly or by being used implicitly as the default (in which case we need to make sure the distribution of `libcurl` and `openssl` that are linked are `conda-forge`'s; I am not even sure of this at the moment).

Reply:
I think this mostly depends on the default value of this config option:
- either it is "system"  and then we want ti to be the MAMBA_ROOT_PREFIX > system approach, and this behavior can be ovveriden by specifying another value for it.
- it is something else, and system can be used to explicitely ignore the MAMBA_ROOT_PREFIX certificates.